### PR TITLE
#572 - read and save groups in AddUpdateUserSlice

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #572 
Read user groups from request json in `AddUpdateUserSlice`

### CODE REVIEW

File: src/main/java/com/artipie/Users.java [hunk 1]
```diff
@@ -188,7 +188,7 @@ public FromStorageYaml(final Storage storage, final Key key) {
                         for (final String group : user.ugroups) {
                             seq = seq.add(group);
                         }
-                        info.add(FromStorageYaml.GROUPS, seq.build());
+                        info = info.add(FromStorageYaml.GROUPS, seq.build());
```
Comment (olegmoz) [inline src/main/java/com/artipie/Users.java:191]: @olenagerasimova it seems we lack test coverage for adding groups, I think we should update `UsersFromStorageYamlTest` to test it

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, I've fixed the test

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#621` is not assigned, can't get performer

**Pull request merged**